docs/user.md

```diff
@@ -282,6 +282,13 @@ source types is given below.
       persistentVolumeClaim:
         claimName: kvc-resource-a150fd63-11c4-11e8-8397-0a580a440340
   ```
+  ### Caveats ###
+    Please keep in mind that during Persistent Volume creation, Kubernetes does not validate the `server` or `path` provided by configuration file and for that reason `KVC` follows the same practice.
```

Comment:
We can just say here: `The NFS server ip and path are not validated, so please ensure that the servers are routable and paths are valid prior to the creation of the VolumeManager CR.`